assets/js/cell/live_editor/theme.js

```diff
@@ -54,7 +57,7 @@ const theme = {
   ],
 
   colors: {
-    "editor.background": "#282c34",
+    "editor.background": settings.editor_background_color,
```

Comment:
This won't update when the user changes the color. I think we *should* have a separate theme and the background synchronization as an addition to that. You can most likely define the high contrast theme as inheriting from the main one, so you only need to override the background (and maybe text) :)